docs/interactions/Message_Components.md

```diff
@@ -14,28 +14,33 @@ The top-level `components` field is an array of [Action Row](#DOCS_INTERACTIONS_
+| title?       | string                                                                                                      | title of the modal                                                                        | [Interaction Callback Type Modal](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object-interaction-callback-type)                                                                                                                                  |
+| min_length?  | int                                                                                                         | the minimum length of the text input                                                      | [Input Text](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text)                                                                                                                                                                                                        |
+| max_length?  | int                                                                                                         | the maximum length of the text input                                                      | [Input Text](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text)                                                                                                                                                                                                        |
+| placeholder? | string                                                                                                      | the placeholder for the text input field                                                  | [Input Text](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/input-text)                                                                                                                                                                                                        |
 
 ###### Component Types
 
-| Type | Name        | Description                            |
-| ---- | ----------- | -------------------------------------- |
-| 1    | Action Row  | A container for other components       |
-| 2    | Button      | A button object                        |
-| 3    | Select Menu | A select menu for picking from choices |
+| Type | Name        | Description                                |
+| ---- | ----------- | ------------------------------------------ |
+| 1    | Action Row  | A container for other components           |
+| 2    | Button      | A button object                            |
+| 3    | Select Menu | A select menu for picking from choices     |
+| 4    | Input Text  | A text box for inserting written responses |
```

Comment:
```suggestion
| 4    | Input Text  | A text box for inserting written responses in modals |
```
Otherwise it looks to me like you can also use these for regular messages

Reply:
Ah yes, thanks!